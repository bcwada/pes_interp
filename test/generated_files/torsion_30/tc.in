spinmult 1
charge 0
basis           def2-svp
coordinates     geom.xyz
guess           sad
method          b3lyp
run             gradient
new_minimizer   true
xtol            0.0001
