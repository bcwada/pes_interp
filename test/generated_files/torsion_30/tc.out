| Startfile from command line: tc.in


|       ***********************************************************
|       *                    TeraChem v1.9-2022.05-dev            *
|       *  Git Version: d4b8df7494a2ef35cff77b6c7632844e0ef90f1f  *
|       *                   Development Version                   *
|       *           Chemistry at the Speed of Graphics!           *
|       ***********************************************************
|       * This program may only be used in connection with        *
|       * a valid license from PetaChem, LLC. Use of this program *
|       * or results thereof indicates acceptance of all terms    *
|       * and conditions stated in the license and that a valid   *
|       * license agreement between the user and PetaChem, LLC    *
|       * exists. PetaChem, LLC does not warrant the correctness  *
|       * of results or their suitability for any purpose.        *
|       * Please email bugs, suggestions, and comments to         *
|       *                  help@petachem.com                      *
|       *                                                         *
|       ***********************************************************


|       ***********************************************************
|       *  Compiled by henryw7      Thu Jun  2 12:35:51 PDT 2022  *
|       *  Supported architectures: sm_35 sm_52 sm_61 sm_70 sm_80 *
|       *  Build cuda_11.4.r11.4/compiler.30188945_0              *
|       ***********************************************************


| Job started   Fri Nov 18 20:18:02 2022
| On fire-11-01 (available memory: 176901 MB)

| ######################################### RUNTIME INFO ##########################################
| terachem  tc.in 

|  NVRM version: NVIDIA UNIX x86_64 Kernel Module  460.32.03  Sun Dec 27 19:00:34 UTC 2020
|  GCC version:  gcc version 4.8.5 20150623 (Red Hat 4.8.5-16) (GCC) 

|  	linux-vdso.so.1 =>  (0x00007fff3b3c1000)
|  	libcurl.so.4 => /usr/lib64/libcurl.so.4 (0x00002acbf8949000)
|  	libmpi.so.12 => /global/software/intel/2017.8.262/compilers_and_libraries_2017.8.262/linux/mpi/intel64/lib/libmpi.so.12 (0x00002acbf8bb2000)
|  	libdftbplus.so => /global/software/TeraChem/2022.06-intel-2017.8.262-CUDA-11.4.1/lib/libdftbplus.so (0x00002acbf98e8000)
|  	libfsockets.so => /global/software/TeraChem/2022.06-intel-2017.8.262-CUDA-11.4.1/lib/libfsockets.so (0x00002acbf9dce000)
|  	libxmlf90.so => /global/software/TeraChem/2022.06-intel-2017.8.262-CUDA-11.4.1/lib/libxmlf90.so (0x00002acbf9fd2000)
|  	libthcbox.so.1 => /global/software/TeraChem/2022.06-intel-2017.8.262-CUDA-11.4.1/lib/libthcbox.so.1 (0x00002acbfa1f5000)
|  	libOpenMM.so => /global/software/OpenMM/7.4.2-intel-2017.8.262-CUDA-11.0.2/lib/libOpenMM.so (0x00002acbfbf42000)
|  	libgpubox.so.1 => /global/software/TeraChem/2022.06-intel-2017.8.262-CUDA-11.4.1/lib/libgpubox.so.1 (0x00002acbfc5a6000)
|  	libtensorbox.so.1 => /global/software/TeraChem/2022.06-intel-2017.8.262-CUDA-11.4.1/lib/libtensorbox.so.1 (0x00002acbfc7b9000)
|  	libiobox.so.1 => /global/software/TeraChem/2022.06-intel-2017.8.262-CUDA-11.4.1/lib/libiobox.so.1 (0x00002acbfc9d5000)
|  	libintbox.so.1 => /global/software/TeraChem/2022.06-intel-2017.8.262-CUDA-11.4.1/lib/libintbox.so.1 (0x00002acbfcbe5000)
|  	libsqmbox.so.1 => /global/software/TeraChem/2022.06-intel-2017.8.262-CUDA-11.4.1/lib/libsqmbox.so.1 (0x00002acc389c2000)
|  	libgridbox.so.1 => /global/software/TeraChem/2022.06-intel-2017.8.262-CUDA-11.4.1/lib/libgridbox.so.1 (0x00002acc38f1a000)
|  	libdftbox.so.1 => /global/software/TeraChem/2022.06-intel-2017.8.262-CUDA-11.4.1/lib/libdftbox.so.1 (0x00002acc39493000)
|  	libcibox.so.1 => /global/software/TeraChem/2022.06-intel-2017.8.262-CUDA-11.4.1/lib/libcibox.so.1 (0x00002acc39775000)
|  	libccbox.so.1 => /global/software/TeraChem/2022.06-intel-2017.8.262-CUDA-11.4.1/lib/libccbox.so.1 (0x00002acc39f46000)
|  	libsimbox.so.1 => /global/software/TeraChem/2022.06-intel-2017.8.262-CUDA-11.4.1/lib/libsimbox.so.1 (0x00002acc3a511000)
|  	liboptbox.so.1 => /global/software/TeraChem/2022.06-intel-2017.8.262-CUDA-11.4.1/lib/liboptbox.so.1 (0x00002acc3a72e000)
|  	libnvToolsExt.so.1 => /global/software/CUDA/11.4.1/lib64/libnvToolsExt.so.1 (0x00002acc3a962000)
|  	libprotobuf.so.25 => /global/software/protobuf/3.14.0/lib/libprotobuf.so.25 (0x00002acc3ab6c000)
|  	libcommbox.so.1 => /global/software/TeraChem/2022.06-intel-2017.8.262-CUDA-11.4.1/lib/libcommbox.so.1 (0x00002acc3b01f000)
|  	libcuda.so.1 => /usr/lib64/libcuda.so.1 (0x00002acc3b284000)
|  	libcudart.so.11.0 => /global/software/TeraChem/2022.06-intel-2017.8.262-CUDA-11.4.1/lib/libcudart.so.11.0 (0x00002acc3c96c000)
|  	libcublas.so.11 => /global/software/TeraChem/2022.06-intel-2017.8.262-CUDA-11.4.1/lib/libcublas.so.11 (0x00002acc3cc0e000)
|  	libcufft.so.10 => /global/software/TeraChem/2022.06-intel-2017.8.262-CUDA-11.4.1/lib/libcufft.so.10 (0x00002acc45b47000)
|  	libcusparse.so.11 => /global/software/TeraChem/2022.06-intel-2017.8.262-CUDA-11.4.1/lib/libcusparse.so.11 (0x00002acc5b7c7000)
|  	libcusolver.so.11 => /global/software/TeraChem/2022.06-intel-2017.8.262-CUDA-11.4.1/lib/libcusolver.so.11 (0x00002acc69bb2000)
|  	libcrypto.so.10 => /usr/lib64/libcrypto.so.10 (0x00002acc76e7b000)
|  	libssl.so.10 => /usr/lib64/libssl.so.10 (0x00002acc772dc000)
|  	libstdc++.so.6 => /usr/lib64/libstdc++.so.6 (0x00002acc7754e000)
|  	libm.so.6 => /usr/lib64/libm.so.6 (0x00002acc77856000)
|  	libcilkrts.so.5 => /global/software/TeraChem/2022.06-intel-2017.8.262-CUDA-11.4.1/lib/libcilkrts.so.5 (0x00002acc77b58000)
|  	libgcc_s.so.1 => /usr/lib64/libgcc_s.so.1 (0x00002acc77d93000)
|  	libpthread.so.0 => /usr/lib64/libpthread.so.0 (0x00002acc77fa9000)
|  	libc.so.6 => /usr/lib64/libc.so.6 (0x00002acc781c5000)
|  	/lib64/ld-linux-x86-64.so.2 (0x00002acbf8725000)
|  	libdl.so.2 => /usr/lib64/libdl.so.2 (0x00002acc78588000)
|  	libidn.so.11 => /usr/lib64/libidn.so.11 (0x00002acc7878c000)
|  	libssh2.so.1 => /usr/lib64/libssh2.so.1 (0x00002acc789bf000)
|  	libssl3.so => /usr/lib64/libssl3.so (0x00002acc78be9000)
|  	libsmime3.so => /usr/lib64/libsmime3.so (0x00002acc78e35000)
|  	libnss3.so => /usr/lib64/libnss3.so (0x00002acc7905c000)
|  	libnssutil3.so => /usr/lib64/libnssutil3.so (0x00002acc79386000)
|  	libplds4.so => /usr/lib64/libplds4.so (0x00002acc795b3000)
|  	libplc4.so => /usr/lib64/libplc4.so (0x00002acc797b7000)
|  	libnspr4.so => /usr/lib64/libnspr4.so (0x00002acc799bc000)
|  	libgssapi_krb5.so.2 => /usr/lib64/libgssapi_krb5.so.2 (0x00002acc79bfa000)
|  	libkrb5.so.3 => /usr/lib64/libkrb5.so.3 (0x00002acc79e47000)
|  	libk5crypto.so.3 => /usr/lib64/libk5crypto.so.3 (0x00002acc7a12f000)
|  	libcom_err.so.2 => /usr/lib64/libcom_err.so.2 (0x00002acc7a362000)
|  	liblber-2.4.so.2 => /usr/lib64/liblber-2.4.so.2 (0x00002acc7a566000)
|  	libldap-2.4.so.2 => /usr/lib64/libldap-2.4.so.2 (0x00002acc7a775000)
|  	libz.so.1 => /usr/lib64/libz.so.1 (0x00002acc7a9c9000)
|  	librt.so.1 => /usr/lib64/librt.so.1 (0x00002acc7abdf000)
|  	libifport.so.5 => /global/software/intel/2017.8.262/compilers_and_libraries_2017.8.262/linux/compiler/lib/intel64/libifport.so.5 (0x00002acc7ade7000)
|  	libifcoremt.so.5 => /global/software/intel/2017.8.262/compilers_and_libraries_2017.8.262/linux/compiler/lib/intel64/libifcoremt.so.5 (0x00002acc7b016000)
|  	libimf.so => /global/software/intel/2017.8.262/compilers_and_libraries_2017.8.262/linux/compiler/lib/intel64/libimf.so (0x00002acc7b3a6000)
|  	libsvml.so => /global/software/intel/2017.8.262/compilers_and_libraries_2017.8.262/linux/compiler/lib/intel64/libsvml.so (0x00002acc7b893000)
|  	libiomp5.so => /global/software/intel/2017.8.262/compilers_and_libraries_2017.8.262/linux/compiler/lib/intel64/libiomp5.so (0x00002acc7c7b1000)
|  	libintlc.so.5 => /global/software/intel/2017.8.262/compilers_and_libraries_2017.8.262/linux/compiler/lib/intel64/libintlc.so.5 (0x00002acc7cb55000)
|  	libirng.so => /global/software/intel/2017.8.262/compilers_and_libraries_2017.8.262/linux/compiler/lib/intel64/libirng.so (0x00002acc7cdc1000)
|  	libcublasLt.so.11 => /global/software/CUDA/11.4.1/lib64/libcublasLt.so.11 (0x00002acc7d134000)
|  	libkrb5support.so.0 => /usr/lib64/libkrb5support.so.0 (0x00002acc8f243000)
|  	libkeyutils.so.1 => /usr/lib64/libkeyutils.so.1 (0x00002acc8f451000)
|  	libresolv.so.2 => /usr/lib64/libresolv.so.2 (0x00002acc8f655000)
|  	libsasl2.so.3 => /usr/lib64/libsasl2.so.3 (0x00002acc8f86f000)
|  	libselinux.so.1 => /usr/lib64/libselinux.so.1 (0x00002acc8fa8c000)
|  	libcrypt.so.1 => /usr/lib64/libcrypt.so.1 (0x00002acc8fcb3000)
|  	libpcre.so.1 => /usr/lib64/libpcre.so.1 (0x00002acc8feea000)
|  	libfreebl3.so => /usr/lib64/libfreebl3.so (0x00002acc9014c000)
| #################################################################################################

| Cannot find license.dat file in the TeraChem installation directory /global/software/TeraChem/2022.06-intel-2017.8.262-CUDA-11.4.1


| TRYING THE NETWORK LICENSE...
| Connecting to license server '54.208.252.40' port '8877'...
| Connected!
| Checking your license...
-----------------------------------
Processed Input file:
                 spinmult 1
                   charge 0
                    basis def2-svp
              coordinates geom.xyz
                    guess sad
                   method b3lyp
                      run gradient
            new_minimizer true
                     xtol 0.0001
-----------------------------------
Jobname: geom
Scratch directory: ./scr.geom
| Random number seed: 1644915287

XYZ coordinates geom.xyz
Molden File Output: ./scr.geom/geom.molden
Using basis set: def2-svp
dmrgstart not found
Spin multiplicity: 1
DIIS will use up to 10 vectors.
Condition number limit for the DIIS overlap matrix is            0
WF convergence threshold: 3.00e-05
Using DIIS algorithm to converge WF
Maximum number of SCF iterations: 100
Incremental fock with rebuild every 8 iterations
Will switch to conventional Fock if diffuse functions are detected
X-matrix tolerance: 1.00e-04
PRECISION: DYNAMIC
TeraChem will select linear algebra engine
DFT Functional requested: b3lyp
Method: B3LYP 
  Hartree-Fock exact exchange:          0.20 
  Slater exchange functional:           0.80 
  Becke 1988 exchange functional:       0.72 
  Lee-Yang-Parr correlation functional: 0.81 
  VWN(I) correlation functional:        0.19 
Wavefunction: RESTRICTED
DFT grid type: 1
Using dynamic DFT grids.
Initial guess generated from superposition of atomic densities generated with HF,
using spherically symmetric atomic densities and averaged alpha and beta spin densities.

********************************************
**** SINGLE POINT GRADIENT CALCULATIONS ****
********************************************
SCF:   Initial guess is taken from previous step
*************WARNING***************
Unrecognized or unnecessary keywords:
These are being ignored.
   new_minimizer true
***********************************
Starting TensorBox...
| TeraChem will select linear algebra engine
| Using cuSolver for matrix diagonalization

| using 1 out of 1 CUDA devices
|   Device 0:      GeForce GTX 970,  4043MB, CC 5.2 -- CPU THREAD 0
| -------------------------------------------------------------------
| CPU Memory Available: 48347.12 MegaWords 
| GPU Memory Available: 249.38 MegaWords 
| Maximum recommended basis set size: 5100 basis functions
|  (limited by GPU memory)
| -------------------------------------------------------------------
| Initialize GPUBox with 128 MW of memory

| **************************************************************
|  Greetings, Martinez Group! You have 1000 licenses in total
|  IN USE: 14
|  AVAILABLE: 986
| **************************************************************



Basis set:       def2-svp
Total atoms:     14
Total charge:    0
Total electrons: 34 (17-alpha, 17-beta)
Number electrons modeled by ECPs: 0
Total orbitals:  110
Total AO shells: 54 (32 S-shells; 18 P-shells; 4 D-shells; 0 F-shells; 0 G-shells)
Spin multiplicity: 1
Nuclear repulsion energy (QM atoms): 133.445393246825 a.u.

Using neutral atomic charges and default spins and spin directions.
Number of unique atoms is: 2 
Atom #1 is unique. Calculating atomic density. 
XCEnergy: 0.000000 NNRepEnergy: 0.000000 
Rest of SCF Energy: -36.042762
XCEnergy: 0.000000 NNRepEnergy: 0.000000 
Rest of SCF Energy: -36.042762
Iteration: 0
XCEnergy: 0.000000 NNRepEnergy: 0.000000 
Rest of SCF Energy: -37.644434
XCEnergy: 0.000000 NNRepEnergy: 0.000000 
Rest of SCF Energy: -37.644434
Iteration: 1
XCEnergy: 0.000000 NNRepEnergy: 0.000000 
Rest of SCF Energy: -37.648420
XCEnergy: 0.000000 NNRepEnergy: 0.000000 
Rest of SCF Energy: -37.648420
Iteration: 2
XCEnergy: 0.000000 NNRepEnergy: 0.000000 
Rest of SCF Energy: -37.648733
XCEnergy: 0.000000 NNRepEnergy: 0.000000 
Rest of SCF Energy: -37.648733
Iteration: 3
XCEnergy: 0.000000 NNRepEnergy: 0.000000 
Rest of SCF Energy: -37.648783
XCEnergy: 0.000000 NNRepEnergy: 0.000000 
Rest of SCF Energy: -37.648783
Iteration: 4
XCEnergy: 0.000000 NNRepEnergy: 0.000000 
Rest of SCF Energy: -37.648784
XCEnergy: 0.000000 NNRepEnergy: 0.000000 
Rest of SCF Energy: -37.648784
Iteration: 5
XCEnergy: 0.000000 NNRepEnergy: 0.000000 
Rest of SCF Energy: -37.648784
XCEnergy: 0.000000 NNRepEnergy: 0.000000 
SCF Energy: -37.648784
Atom #5 is unique. Calculating atomic density. 
XCEnergy: 0.000000 NNRepEnergy: 0.000000 
Rest of SCF Energy: -0.499278
XCEnergy: 0.000000 NNRepEnergy: 0.000000 
Rest of SCF Energy: -0.499278
Iteration: 0
XCEnergy: 0.000000 NNRepEnergy: 0.000000 
Rest of SCF Energy: -0.499278
XCEnergy: 0.000000 NNRepEnergy: 0.000000 
Rest of SCF Energy: -0.499278
Iteration: 1
XCEnergy: 0.000000 NNRepEnergy: 0.000000 
Rest of SCF Energy: -0.499278
XCEnergy: 0.000000 NNRepEnergy: 0.000000 
SCF Energy: -0.499278
Using neutral atomic charges and default spins and spin directions.
Number of unique atoms is: 2 
Atom #1 is unique. Calculating atomic density. 
XCEnergy: 0.000000 NNRepEnergy: 0.000000 
Rest of SCF Energy: -36.042762
XCEnergy: 0.000000 NNRepEnergy: 0.000000 
Rest of SCF Energy: -36.042762
Iteration: 0
XCEnergy: 0.000000 NNRepEnergy: 0.000000 
Rest of SCF Energy: -37.644434
XCEnergy: 0.000000 NNRepEnergy: 0.000000 
Rest of SCF Energy: -37.644434
Iteration: 1
XCEnergy: 0.000000 NNRepEnergy: 0.000000 
Rest of SCF Energy: -37.648420
XCEnergy: 0.000000 NNRepEnergy: 0.000000 
Rest of SCF Energy: -37.648420
Iteration: 2
XCEnergy: 0.000000 NNRepEnergy: 0.000000 
Rest of SCF Energy: -37.648733
XCEnergy: 0.000000 NNRepEnergy: 0.000000 
Rest of SCF Energy: -37.648733
Iteration: 3
XCEnergy: 0.000000 NNRepEnergy: 0.000000 
Rest of SCF Energy: -37.648783
XCEnergy: 0.000000 NNRepEnergy: 0.000000 
Rest of SCF Energy: -37.648783
Iteration: 4
XCEnergy: 0.000000 NNRepEnergy: 0.000000 
Rest of SCF Energy: -37.648784
XCEnergy: 0.000000 NNRepEnergy: 0.000000 
Rest of SCF Energy: -37.648784
Iteration: 5
XCEnergy: 0.000000 NNRepEnergy: 0.000000 
Rest of SCF Energy: -37.648784
XCEnergy: 0.000000 NNRepEnergy: 0.000000 
SCF Energy: -37.648784
Atom #5 is unique. Calculating atomic density. 
XCEnergy: 0.000000 NNRepEnergy: 0.000000 
Rest of SCF Energy: -0.499278
XCEnergy: 0.000000 NNRepEnergy: 0.000000 
Rest of SCF Energy: -0.499278
Iteration: 0
XCEnergy: 0.000000 NNRepEnergy: 0.000000 
Rest of SCF Energy: -0.499278
XCEnergy: 0.000000 NNRepEnergy: 0.000000 
Rest of SCF Energy: -0.499278
Iteration: 1
XCEnergy: 0.000000 NNRepEnergy: 0.000000 
Rest of SCF Energy: -0.499278
XCEnergy: 0.000000 NNRepEnergy: 0.000000 
SCF Energy: -0.499278
Setting up the DFT grid...
| time to set the grid = 0.00 s
DFT grid points: 13746 (981 points/atom)
Setting up the DFT grid...
| time to set the grid = 0.01 s
DFT grid points: 38226 (2730 points/atom)

                      *** Start SCF Iterations ***

 Iter     DIIS Error       Energy change      Electrons        XC Energy           Energy        Time(s)
----------------------------------------------------------------------------------------------------------
| Purifying guess...Density matrix is not well-conditioned, rescale and purify
| Purification run on CPU (double precision)  using TC2 purification.
| Density matrix purification finished after a total of 18 matrix multiplies
| THRESPDP set to 1.00e+00
|                 >>> SWITCHING TO GRID 1 <<<
|    1     0.4878032277    -157.5145522338    34.0226919739   -19.8572725071    -157.5145522338     0.15  
|                 >>> SWITCHING TO GRID 0 <<<
|    2     0.2862445738      -0.2140414150    33.9996721401   -21.2489248157    -157.7285936488     0.10  
|    3     0.1086000080      -0.5045695730    34.0000967658   -20.2769955887    -158.2331632217     0.10  
|    4     0.0080748764      -0.1032181461    34.0000791678   -20.5240589865    -158.3363813678     0.10  
|                 >>> SWITCHING TO GRID 1 <<<
|    5     0.0028880463      -0.0008643679    33.9997745048   -20.5281655213    -158.3372457357     0.15  
|    6     0.0010413754      -0.0000496545    33.9997755171   -20.5279337090    -158.3372953902     0.15  
| THRESPDP set to 3.07e-02
|    7     0.0001396299      -0.0000092997    33.9997762817   -20.5280023212    -158.3373046899     0.18  
|    8     0.0000239876      +0.0000001729    33.9997764734   -20.5281460745    -158.3373045170     0.19  
----------------------------------------------------------------------------------------------------------
FINAL ENERGY: -158.3373045170 a.u.
HOMO-LUMO GAP: 9.708 eV
CENTER OF MASS: {-0.365659, -0.262726, 0.292200} ANGS
DIPOLE MOMENT: {0.024724, -0.012885, -0.014099} (|D| = 0.031242) DEBYE

Running Mulliken population analysis...


Gradient units are Hartree/Bohr
---------------------------------------------------
        dE/dX            dE/dY            dE/dZ
   -0.0041233584    -0.0089509673     0.0033586675
   -0.0066981045    -0.0049957967    -0.0005976424
    0.0019778040     0.0074709613     0.0032135135
    0.0023375116     0.0098860562    -0.0023157698
    0.0034766602     0.0027179899    -0.0036689028
    0.0024339856    -0.0037416285     0.0064930729
   -0.0052565280    -0.0031154359    -0.0040814036
    0.0035211000    -0.0008802942     0.0072227527
   -0.0021902967    -0.0015121875    -0.0055060489
    0.0043889003     0.0003883171     0.0042850609
   -0.0036736836     0.0009053985    -0.0071178936
    0.0036590513     0.0039815403     0.0049743712
    0.0017504303    -0.0054244867     0.0007154979
   -0.0016034669     0.0032705326    -0.0069752760
---------------------------------------------------
Net gradient:  5.154793e-09 -9.164504e-10 -5.868474e-10
Net torque:    4.459409e-05  4.211677e-05 -1.862763e-04

Writing out molden info
| Total processing time: 1.85 sec 

| Job finished: Fri Nov 18 20:18:06 2022

